src/PoolBuffer.h

```diff
@@ -77,25 +83,29 @@ class ChanData
 class StdDev
 {
    public:
-    StdDev(int w);
+    StdDev(int w, int id);
     void reset();
-    void tick();
+    double tick();
     QElapsedTimer* mTimer;
```

Comment:
If you remove the asterisk, you shouldn't need any new.

Reply:
done, good catch here, too

Reply:
Thanks!